Bumerang/app/src/main/java/com/seng480b/bumerang/fragments/CreateRequestFragment.java

```diff
@@ -365,16 +379,29 @@ private Request createRequest() {
         }
     }
 
-    private void postToFacebook(String titleStr, String descriptionStr, int hoursInt, int minutesInt) {
+    private void postToFacebook(boolean borrow,String titleStr, String descriptionStr, int hoursInt, int minutesInt) {
         FacebookSdk.sdkInitialize(getActivity().getApplicationContext());
         ShareDialog shareDialog = new ShareDialog(getActivity());
 
-        String quote = "Hey everybody, I'm currently looking for a " + titleStr + ".";
-        if (descriptionStr.equals("")) {
-            quote = quote + " I need it within " + String.valueOf(hoursInt + ((double) minutesInt / 60)) + " hours please!";
-        } else {
-            quote = quote + " I would describe it as a " + descriptionStr + ". I need it within " +
-                    String.valueOf(hoursInt + ((double) minutesInt / 60)) + " hours please!";
+        String quote = "";
+
+        if (borrow==true) {
```

Comment:
then just check if requestType is = borrow, or lend. We have constants for them somewhere haha.